program/src/state.rs

```diff
@@ -756,27 +757,13 @@ impl PubkeyAndEntry<Validator> {
         program_id: &Pubkey,
         solido_account: &Pubkey,
         seed: u64,
+        stake_type: &StakeType,
```

Comment:
We can take it by value and derive `Copy` for `StakeType`, it should not take more than a byte, and it makes the call sites a little less cluttered because we can avoid the `&`.